src/components/backend-ai-summary-view.ts

```diff
@@ -358,8 +357,8 @@ export default class BackendAISummary extends BackendAIPage {
       this.invitations = res.invitations;
       if (this.active && !refreshOnly) {
         setTimeout(() => {
-          this._refreshInvitations()
-        }, 15000);
+          this._refreshInvitations();
+        }, 1000);
```

Comment:
Folder invitation is not a frequent event. Do we need to refresh the invitation for 1 s?

Reply:
Well, as I mentioned in the other PR, I guess that It could be handled by #772.
Anyway, 1 sec is way too short, as you commented, so I'll change it.
Btw I presume that 15 seconds it quite long, would it be Okay to reduce the refreshing time such as 10 seconds or less?

Reply:
Okay, then let's go with 10 s for now.